### DISCUSSION TITLE
How to integrated custom Mozilla Hubs client in react App

### DISCUSSION THREAD

**Discussion by BhaveshKumawat008** [mozilla/hubs#5362] [Hubs Cloud Questions] [answered]
Hello experts,

Is any possible way for integrated custom Mozilla hubs client in react app? 

If yes, then Please can share any example/site related to this.

Appreciate any help

**Comment by matthewbcool** [comment, marked as answer]
Hello! Can you tell us more about your goals? It's possible to extend hubs in a variety of ways. Here are a few examples: https://hubs.mozilla.com/labs/extending-hubs-with-hubs-cloud/
